mod/metrics/lib/javascript/script_metrics.js.coffee

```diff
@@ -49,18 +72,40 @@ decko.editorContentFunctionMap['.wikiRating-editor'] = ->
 
 wikiRatingEditorHash = (table) ->
   hash = {}
-  table.find("tbody tr").each ->
+  variableMetricRows(table).each ->
     tr = $(this)
     if key = tr.find(".metric-label .thumbnail").data "cardName"
       hash[key] = tr.find(".metric-weight input").val()
-  hash
+  hash 
+
+# if all values are equals active the equalize
+activeEqualize = () -> 
+  if !getValuesFromTable( $('.wikiRating-editor') )  
+    $('#equalizer').prop('checked', false)
+  else 
+    $('#equalizer').prop('checked', true)
+    toEqualize( $('.wikiRating-editor') )
+
+toEqualize = (table) -> 
+  val = (100 / (variableMetricRows(table).length - 1)).toFixed(2)
+    
+  variableMetricRows(table).each ->
+    tr = $(this)
+    tr.find('td.metric-weight').find('input').val(val)
```

Comment:
this also looks like it should be a separate method to me.  like setAllVariableValuesTo(table, val)